app/assets/javascripts/uploadcare/settings.coffee

```diff
@@ -37,6 +37,8 @@ uploadcare.namespace 'settings', (ns) ->
     cdnBase: 'https://ucarecdn.com'
     urlBase: 'https://upload.uploadcare.com'
     socialBase: 'https://social.uploadcare.com'
+    previewBase: null
```

Comment:
if this is used to get the URL for file preview (afaiu), the name should reflect that: `previewUrlProvider` or something.

Reply:
From Widget point of view, this is exactly the base which is used for preview URLs. It doesn't use to get some other URL, it used as part of the URL itself. So `previewBase` perfectly reflects this.